projects/ng-live-docs/src/documentation.ts

```diff
@@ -92,11 +93,31 @@ export const Documentation = {
      * Returns angular routes used when displaying the documentation/examples for the components.
      */
     getRoutes(): Routes {
-        return Documentation.getAllEntries().map((documentationEntry: DocumentationEntry) => ({
+        let routes = Documentation.getAllEntries().map((documentationEntry: DocumentationEntry) => ({
             path: documentationEntry.urlSegment,
             component: DocumentationContainerComponent,
             data: { documentationEntry },
+            // Add children if there are multi-examples.
+            children: documentationEntry.examples && documentationEntry.examples.length ? [
+                ...documentationEntry.examples.map((exampleEntry: ExampleEntry) => ({
+                    path: exampleEntry.component.name.toLowerCase(),
+                    component: ExampleViewerComponent,
+                    data: { exampleEntry }
+                }))
+            ] : []
         }));
+        const redirectList = [];
+        // Add redirect routes.
+        routes.forEach(route => {
```

Comment:
Usually, if you create an empty array and populate it in a loop, you can replace it with array processing functions.

TypeScript now has a ? so you don't need to do `if (a && a.b && a.b.c)`, you can just do `if (a?.b?.c)`
We also prefer template strings over string concatenation

```ts
const redirectList = routes.filter(route => route.children?.length > 1).map(route => ({
    path: route.data.documentationEntry.urlSegment,
    redirectTo: `${route.data.documentationEntry.urlSegment}/${route.children[0].path}
}));

```